### PR TITLE
Added intro to rate limiting plugin

### PR DESCRIPTION
**Pull request by UkiahSmith**
Soft intro to help people better understand that the EE rate limiting plugin is different than the CE version.

### CODE REVIEW

File: app/docs/enterprise/0.31-x/plugins/rate-limiting.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: Rate Limiting Plugin
 ---
 # Rate Limiting Plugin
+
+The enterprise rate limiting plugin makes a few changes that allow for several benefits over the community version. These changes require a little bit of consideration when configuring and deploying the enterprise rate limiting plugin. Take a moment to read over this Kong blog post on [designing a scalable rate limiting algorithm](https://konghq.com/blog/how-to-design-a-scalable-rate-limiting-algorithm/) and the docs on the underlying [rate limiting library]({% post_url /docs/enterprise/0.31-x/plugins/rate-limiting %}) to familiarize yourself with the underlying differences.
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/plugins/rate-limiting.md]: The URL `docs/enterprise/0.31-x/plugins/rate-limiting` should be `docs/enterprise/latest/plugins/rate-limiting`

**Comment by UkiahSmith** [comment]
Updated PR

included additional configuration options not previously mentioned.

**Comment by coopr** [comment]
@UkiahSmith can this PR get moved along? That'd be great!

File: app/enterprise/0.32-x/plugins/rate-limiting-advanced.md [hunk 2]
```diff
@@ -43,11 +64,14 @@ $ curl -i -X POST http://kong:8001/plugins \
...
+|`config.window_type`| `sliding` | This sets the time window to either `sliding` or `fixed`. Sliding windows factor in weighted request counts from previous time segments when determining whether a client has exceeded its limit, whereas fixed windows only count requests during the current time segment.
+|`config.hide_client_headers`| `false` | Controls whether `X-Ratelimit-Remaining` and `X-Ratelimit-Limit` headers are sent to clients. When set to `true`, headers are not sent/are hidden.
 
 **Note:  Redis configuration values are ignored if the "cluster" strategy is used.**
 
-**Note: PostgreSQL 9.5+ is required when using the "cluster" strategy with "postgres" as the backing Kong cluster data store. This requirement varies from the PostgreSQL 9.4+ requirement as described in the <a href="/install/source">Kong Community Edition documentation</a>.**
+**Note: PostgreSQL 9.5+ is required when using the "cluster" strategy with "postgres" as the backing Kong
+cluster data store. This requirement varies from the PostgreSQL 9.4+ requirement as described
+in the <a href="/install/source">Kong Community Edition documentation</a>.**
```
Comment (thibaultcha) [inline app/enterprise/0.32-x/plugins/rate-limiting-advanced.md]: CE also requires PostgreSQL 9.5 now

**Pull request closed without merging**